CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # v2.5.0
```

Comment:
There should be a line for the HMAC signature in the changelog.

Reply:
Added changelong entry - 379d0cf